### DISCUSSION TITLE
Account Activation Link KO

### DISCUSSION THREAD

**Discussion by crendrik** [docmost/docmost#284] [Help / Questions] [answered]
Hello, 

In the email sent to the user for account activation, there is an error. I have two slashes ("//") --> https://mysite//invites/...
How to fix this address?

Thanks

**Comment by Philipinho** [comment, marked as answer]
Make sure your defined `APP_URL` does not have a forward slash.

**Reply by crendrik** [reply]
Thanks, it's ok
